src/document-click.js

```diff
@@ -64,8 +64,8 @@ export class ClickHandler {
     /** @private @const {boolean} */
     this.isIosSafari_ = platform.isIos() && platform.isSafari();
 
-    // Only intercept clicks when iframed.
-    if (this.viewer_.isIframed() && this.viewer_.isOvertakeHistory()) {
+    // Only intercept clicks when embedded.
+    if (this.viewer_.isEmbedded() && this.viewer_.isOvertakeHistory()) {
```

Comment:
@dvoytenko do we want to intercept clicks in case of 
- webview mode
- multi-doc mode?